Jenkinsfile.integration

```diff
@@ -218,7 +217,6 @@ pipeline {
             }
             when {
                 allOf {
-                    expression { env.run_tempest == 'true' }
```

Comment:
We can probably remove the conditional below: should we test osh, we would still need to use tempest.

Reply:
good point, although it may not be ready yet for osh, which would require more changes to it to account for the differences? Let me try to test it if engcloud permits

Reply:
Yes it's probably broken, but it gives us an incentive to fix :)

Reply:
good point, done!